cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -41,6 +41,18 @@ var moveTests = []struct {
 	wantMoveGroups [][]string
 	wantErr        bool
 }{
+	{
+		name: "Custom object with force-move label",
```

Comment:
It is a little weird to have a test moving a single object a no cluster.
Should we make this "Cluster and external Object" like you did in "objectgraph_test.go"?

Reply:
Fixed!